project.clj

```diff
@@ -17,6 +17,11 @@
                  [org.clojure/tools.logging "0.3.1"]
                  [org.clojure/data.codec "0.1.0"]]
 
+  ; When running the tests as `lein test2junit`, emit XUNIT test reports to enable CircleCI
+  ; to collect statistics over time
+  :plugins [[test2junit "1.1.2"]]
+  :test2junit-output-dir "target/test2junit"
```

Comment:
Hello Ivan, I had something very similar to what you are proposing on my first attempt. Then I realized that the documentation mentioning `CIRCLE_TEST_REPORTS` (see https://github.com/kimh/circleci-build-recipies/tree/clojure-test-metadata-with-test2junit) is outdated, it works with CircleCI 1.0, while we are using 2.0 (and 1.0 will be unsupported end of Aug 2018).

So let me see if I understand correctly: with the PR as-is you are already getting test details as in the 4 links in the first comment of this PR. What you would like is to store the XUNIT test output as an artifact, so that you can consult it offline ? I know that CircleCI supports this, but I wanted to be sure this is what you want.

Reply:
@marco-m well, I just want to achieve the following: if the tests are run on CI, the reports should be stored in a special CI's folder that can be referenced with a special env variable. The profit of it is CI stores those artefacts and you may download them any time you want.